source.mak

```diff
@@ -184,6 +184,7 @@ PARSER_SRCS =				\
 	parsers/pythonloggingconfig.c	\
 	parsers/r.c			\
 	parsers/rexx.c			\
+	parsers/robot.c			\
```

Comment:
In addition you may have to update win32/ctags_vs2013.vcxproj  and win32/ctags_vs2013.vcxproj.filters to build ctags on windowns.  I don't know the files well but to find where you have to do update is not difficult.